cli/tests/integration_tests.rs

```diff
@@ -70,6 +70,30 @@ fn eval_p() {
   assert_eq!("3", stdout_str);
 }
 
+#[test]
+
+fn run_from_stdin() {
+  let mut deno = util::deno_cmd()
+    .current_dir(util::root_path())
+    .arg("run")
+    .arg("-")
+    .stdout(std::process::Stdio::piped())
+    .stdin(std::process::Stdio::piped())
+    .spawn()
+    .unwrap();
+  deno
+    .stdin
+    .as_mut()
+    .unwrap()
+    .write_all(b"console.log(\"Hello World\");")
+    .unwrap();
+  let output = deno.wait_with_output().unwrap();
```

Comment:
Run the same command again in this test case but with different input - just to make sure TS cache invalidation works properly